status-im/status-react
# [#15024] Profile password screen

[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #15024

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
Add the new [password screen](https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=2241-251839&t=mrMsNJyXZLYUCGKW-4) and the main behavior/flow to validate the password.

![image](https://user-images.githubusercontent.com/90291778/225995229-42c64632-f27f-409f-bb8a-bd719f545f21.png) ![image](https://user-images.githubusercontent.com/90291778/225995248-560c2ef1-fc1e-4eb4-bc5c-67a2333852a3.png)
![image](https://user-images.githubusercontent.com/90291778/225995311-6fa6b81c-bb21-4f3c-ad40-9a25a7e0e4ab.png) ![image](https://user-images.githubusercontent.com/90291778/225995346-1746abab-2f6f-4b04-a7cb-8dd9a4f2fd48.png)
![image](https://user-images.githubusercontent.com/90291778/225995522-20545735-71eb-4813-aa9f-752e9a1dc8a7.png)


### Review notes
- The disclaimer component don't have a blur variant, it'll be addressed in a following issue
- Please test on small screen devices

### Testing notes
Please test the password validation and input behavior.

#### Platforms

- Android
- iOS

#### Areas that maybe impacted
 Onboarding

### Steps to test

- Open Status in a Fresh install
- Press I'm new to status -> Generate keys -> Continue
- Password validation & inputs should behave as expected, when all the validations are passed, the `confirm password` button should be enabled and the user can continue with the flow.


status: ready
